modules/router/templates/assets_origin.conf.erb

```diff
@@ -38,16 +38,32 @@ server {
   add_header "Access-Control-Allow-Methods" "GET, OPTIONS";
   add_header "Access-Control-Allow-Headers" "origin, authorization";
 
-  <%- @asset_routes.each do |alias_path, vhost_name| -%>
-  location <%= alias_path %> {
-    proxy_set_header Host <%= vhost_name %>.<%= @app_domain %>;
-    proxy_pass <%= @enable_ssl ? 'https' : 'http' %>://<%= vhost_name %>.<%= @app_domain %>;
-  }
-
-  <%- end -%>
-
-  location / {
-    proxy_set_header Host static.<%= @app_domain %>;
-    proxy_pass <%= @enable_ssl ? 'https' : 'http' %>://static.<%= @app_domain %>;
-  }
+  <%- unless scope.lookupvar('::aws_migration') %>
+    <%- @asset_routes.each do |alias_path, vhost_name| -%>
+    location <%= alias_path %> {
+      proxy_set_header Host <%= vhost_name %>.<%= @app_domain %>;
+      proxy_pass <%= @enable_ssl ? 'https' : 'http' %>://<%= vhost_name %>.<%= @app_domain %>;
```

Comment:
could we move the `::aws_migration` look up to just wrap these proxy_pass lines to make it clearer what the difference is?